concepticondata/conceptlists/Lastra-1986-381.tsv

```diff
@@ -0,0 +1,382 @@
+ID	NUMBER	GLOSS	CONCEPTICON_ID	CONCEPTICON_GLOSS
```

Comment:
The ID columns needs to be of the form `LastName-Year-TotalItems-Current-Items`, i.e. `Lastra-1986-381-1`, `Lastra-1986-381-2`, etc.